### ISSUE TITLE
search word (non-latin) is not shown in the 'Search Results' window

### ISSUE THREAD

**Issue by eLiNK2gl** [koreader/kindlepdfviewer#216]
while searching for file names using non-latin alphabet (e.g. Russian), the search word is not shown in the 'Search Results' window.
Only blanks are shown.

**Comment by ghost** [comment]
Changing tfont (in kpdf-settings) by something that supports cyrillic glyphs (say, ["tfont"] = "droid/DroidSansBold.ttf") should help гиганту мысли. BTW, not only search patterns, but also the path (in the header of filechooser.lua) that includes cyrillic characters should be properly shown with a proper font.

**Comment by eLiNK2gl** [comment]
Indeed. But standing on the shoulders of real giants I would expect a more general solution :)
Provided there is a  font that comprises all non-latin characters (?).
